composer/utils/misc.py

```diff
@@ -100,3 +112,37 @@ def using_torch_2_0_1() -> bool:
         bool: Return True if current version is greater than or equal to 2.0.1 else False
     """
     return version.parse(torch.__version__) >= version.parse('2.0.1')
+
+
+def validate_save_planner(save_planner: Optional[Any]) -> None:
+    """Checks that ``save_planner`` is an instance of a :class:`~torch.distributed.checkpoint.planner.SavePlanner`.
+
+    Raises:
+        ValueError: If ``save_planner`` is not a :class:`~torch.distributed.checkpoint.planner.SavePlanner`.
+    """
+    from torch.distributed.checkpoint.planner import SavePlanner
+
+    if save_planner is not None and not isinstance(save_planner, SavePlanner):
+        raise ValueError(
+            (
+                f'save_planner {type(save_planner)} is not a '
+                'torch.distributed.checkpoint.planner.SavePlanner'
+            )
+        )
+
+
+def validate_load_planner(load_planner: Optional[Any]) -> None:
```

Comment:
Can we in-line given it's an if statement + a value error and used in one place?